docs/recording_provenance.md

```diff
@@ -875,3 +875,153 @@ and signing.
+  `[path-to-json-file]`: the path to the JSON-LD file containing the Chronicle Operations.
+
+- If the data is successfully imported, the Chronicle Operations will be added
+  to the Chronicle database.
+
+For example, suppose you want to create a new agent with the name `testagent`
+and a new namespace with the name `testns`. You can add the following
+operations to your JSON-LD file:
+
+```json
+[
+    {
+        "@id": "_:n1",
+        "@type": [
+        "http://btp.works/chronicleoperations/ns#SetAttributes"
+        ],
+        "http://btp.works/chronicleoperations/ns#agentName": [
+        {
+            "@value": "testagent"
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#attributes": [
+        {
+            "@type": "@json",
+            "@value": {}
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#domaintypeId": [
+        {
+            "@value": "type"
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#namespaceName": [
+        {
+            "@value": "testns"
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#namespaceUuid": [
+        {
+            "@value": "6803790d-5891-4dfa-b773-41827d2c630b"
+        }
+        ]
+    },
+    {
+        "@id": "_:n1",
+        "@type": [
+        "http://btp.works/chronicleoperations/ns#CreateNamespace"
+        ],
+        "http://btp.works/chronicleoperations/ns#namespaceName": [
+        {
+            "@value": "testns"
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#namespaceUuid": [
+        {
+            "@value": "6803790d-5891-4dfa-b773-41827d2c630b"
+        }
+        ]
+    },
+    {
+        "@id": "_:n1",
+        "@type": [
+        "http://btp.works/chronicleoperations/ns#AgentExists"
+        ],
+        "http://btp.works/chronicleoperations/ns#agentName": [
+        {
+            "@value": "testagent"
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#namespaceName": [
+        {
+            "@value": "testns"
+        }
+        ],
+        "http://btp.works/chronicleoperations/ns#namespaceUuid": [
+        {
+            "@value": "6803790d-5891-4dfa-b773-41827d2c630b"
+        }
+        ]
+    }
+]
+```
+
+To import data into Chronicle using the `import` command, you need to provide
+the `namespace-id` and `namespace-uuid` arguments. These arguments are required
+to specify the namespace to which the imported data will be imported.
+
+The `namespace-id` argument is a string that specifies the name of the namespace
+to which the data will be imported. This value should match the value of the
+`http://btp.works/chronicleoperations/ns#namespaceName` property in the JSON-LD
+file.
+
+The `namespace-uuid` argument is a string that specifies the UUID of the
+namespace to which the data will be imported. This value should match the value
+of the `http://btp.works/chronicleoperations/ns#namespaceUuid` property in the
+JSON-LD file.
```

Comment:
People might wonder, if they should match, why supply them at all, instead of having the importer just read them from the file. (Just a thought, feel free to summarily resolve for this PR.)